packages/slider/src/Slider.ts

```diff
@@ -410,6 +410,12 @@ export class Slider extends SizedMixin(ObserveSlotText(SliderHandle, ''), {
         `;
     }
 
+    private renderHandle(): TemplateResult {
+        return html`
+            ${this.variant === 'tick' ? html`` : this.handleController.render()}
+        `;
```

Comment:
At some point we should do focused testing on whether the use of stacked templates here causes us any performance costs, but the small scale use here doesn't seem to be tweaking the performance of the component itself, so :shipit: